websites/E/EternalNetwork TM/presence.ts

```diff
@@ -79,16 +79,16 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/memberlist.php"))
-				presenceData.state = "Sneaking into member list !";
+				presenceData.state = "Sneaking into member list";
 
 			if (search.includes("?mode=team"))
-				presenceData.state = "Checking out team list !";
+				presenceData.state = "Checking out team list";
```

Comment:
You might as well use the verb to view instead for all these checking verbs lol

Reply:
Message changed.